### PR TITLE
Change "rocksdb" commandline parameter to a more generic "database"

### PR DESCRIPTION
**Pull request by DarkEld3r**
I'm not sure how many details of underlying database the user should now, but I like a more generic interface, because it will give us more possibilities to change the implementation without additional "breaking changes".

Opinions?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - The `Node` constructor now requires `db` and `services` variables instead of `blockchain` instance. (#366)
 - The `Blockchain` constructor now requires services keypair and an `ApiSender` instance. (#366)
 - `mount_*_api` methods in `Blockchain` instance now do not require `ApiContext`. (#366)
+- Replaced `rocksdb` commandline parameter to more generic `database`.
```
Comment (vldm) [inline CHANGELOG.md]: Add pr number.

File: exonum/src/helpers/fabric/details.rs [hunk 2]
```diff
@@ -87,11 +89,11 @@ impl Command for Run {
...
             ),
             Argument::new_named(
-                "ROCKSDB_PATH",
+                DATABASE_PATH,
                 true,
-                "Use rocksdb database with the given path.",
+                "Use database with the given path.",
                 "d",
-                "rocksdb",
+                "database",
```
Comment (vldm) [inline exonum/src/helpers/fabric/details.rs]: Just a nitpick: May be beter `long_name` will be: "database_path" ?

Comment (aleksuss) [reply exonum/src/helpers/fabric/details.rs]: Underscores in parameters look disgusting as for me. What about `db-path` ?

File: exonum/src/helpers/fabric/details.rs [hunk 3]
```diff
@@ -38,6 +38,8 @@ use super::shared::{AbstractConfig, NodePublicConfig, SharedConfig, NodePrivateC
                     CommonConfigTemplate};
 use super::DEFAULT_EXONUM_LISTEN_PORT;
 
+const DATABASE_PATH: &str = "DATABASE_PATH";
```
Comment (defuz) [inline exonum/src/helpers/fabric/details.rs]: I think the explicit constant is redundant.

File: CHANGELOG.md [hunk 4]
```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `mount_*_api` methods in `Blockchain` instance now do not require `ApiContext`. (#366)
 - Removed redundant `current_height` method in `Schema` and rename `last_height` to `height`. (#379)
 - `last_block` now returns `Block` instead of `Option<Block>`. (#379)
+- Replaced `rocksdb` commandline parameter to more generic `database`. (#376)
```
Comment (defuz) [inline CHANGELOG.md]: `database` or `db-path`?

File: exonum/src/helpers/fabric/details.rs (hunk 3 above)
Comment (DarkEld3r) [reply exonum/src/helpers/fabric/details.rs]: Why not? It is used three times. 😄

Comment (vldm) [reply exonum/src/helpers/fabric/details.rs]: One reason for using constant: rust will statically check typo.

**Pull request merged**